.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: go
 sudo: required
-dist: trusty
+dist: xenial
 
 go:
   - "1.10.2"
 
 go_import_path: github.com/kubernetes-sigs/cri-tools
 
-before_install:
-  - sudo env PATH=$PATH GOPATH=$GOPATH hack/install-cni.sh
-  - sudo env PATH=$PATH GOPATH=$GOPATH hack/install-pouch.sh
```

Comment:
This step will run before each stage. Some stage don't need it.

Reply:
👌